dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -99,3 +99,6 @@ enableImplicitBeanArchives.conflict.CWOWB1017W=CWOWB1017W: The enableImplicitBea
 enableImplicitBeanArchives.conflict.CWOWB1017W.explanation=Although you can set the enableImplicitBeanArchives attribute on both of the cdi12 and cdi configuration element configuration elements, you should not set this attribute with different values in the same server configuration.
 enableImplicitBeanArchives.conflict.CWOWB1017W.useraction=Update your server.xml to fix the conflict. The recommended action is to remove the cdi12 configuration element because it is superseded by the cdi configuration element.
 
+unversioned.beansXML.warning.CWOWB1018W=CWOWB1018W: Using an unversioned CDI beans.xml file may lead to unexpected behaviour when using CDI 4.0 or higher: {0}
```

Comment:
@OpenLiberty/message-reviewer please review this new message

Reply:
Use "might" instead of "may". It is preferable to use "later" instead of "higher" ("CDI 4.0 or later").